src/test/java/org/javalessons/basic/GoogleSearch.java

```diff
@@ -0,0 +1,167 @@
+package org.javalessons.basic;
+
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+
+import java.text.NumberFormat;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.NoSuchElementException;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class GoogleSearch {
+
+    public static final String HTTPS_WWW_AMAZON_DE = "https://www.amazon.de/";
+    public static final String QUERY_AMAZON = "buecher";
+    public static final NumberFormat format = NumberFormat.getInstance(Locale.FRANCE);
+    public static final int PRICE_LIMIT = 10;
+    public static final String HTTPS_WWW_GOOGLE_COM = "https://www.google.com/";
+    public static final String QUERY_TELRAN = "telran berlin";
+    public static final String TEL_RAN_DE = "tel-ran.de";
+
+    WebDriver driver;
+    WebElement element;
+    WebElement element2;
+    private TimeUnit SECONDS;
+    WebDriverWait wait;
+    @Before
+    public void initialize() {
+        String chromeDriverPath = "C:\\Users\\Gainulina\\chromedriver.exe";
+        System.setProperty("webdriver.chrome.driver", chromeDriverPath);
+
+    }
+
+/*
+
+        @Test
+        public void testGoogleSearchWorks() {
```

Comment:
Commenting out code is bad idea. I'd rather remove it if not used or in this case - would put `@Ignore` annotation https://github.com/junit-team/junit4/wiki/ignoring-tests